visualization/app/codeCharta/ui/codeMap/codeMap.preRender.service.ts

```diff
@@ -95,20 +95,17 @@ export class CodeMapPreRenderService implements StoreSubscriber, MetricServiceSu
 	}
 
 	private decorateIfPossible() {
-		if (
-			this.unifiedMap &&
-			this.storeService.getState().files.fileStatesAvailable() &&
-			this.unifiedFileMeta &&
-			this.metricService.getMetricData()
-		) {
+		const store = this.storeService.getState()
```

Comment:
rename to state. There is a difference between state and store. state is just an object while the store provides methods such as dispatch etc